src/test/resources/config/application.yml

```diff
@@ -2,6 +2,7 @@ artemis:
     repo-clone-path: ./repos/
     repo-download-clone-path: ./repos-download/
     encryption-password: <encrypt-password>     # arbitrary password for encrypting database values
+    external-user-management: true
```

Comment:
this is still outdated, please update the file accordingly

Reply:
Updated